src/main/java/se/sundsvall/webmessagecollector/integration/opene/OpenEClient.java

```diff
@@ -34,17 +33,16 @@ byte[] getMessages(String familyId, String fromDate, String toDate) throws IOExc
         return getBytes(buildUrl(properties.getMessagesPath(), Map.of("familyid", familyId), fromDate, toDate));
         
     }
-    
-    private byte[] getBytes(URI url) throws IOException {
-        
-        
-        var result = response.execute(new HttpGet(url));
-        
+
+    private byte[] getBytes(URI url) throws IOException, OpenEException {
+
+        var result = response.execute(new HttpGet(url), response1 -> response1);
+
         if (result.getCode() == 200) {
             return EntityUtils.toByteArray(result.getEntity());
```

Comment:
Does this work? Are there any IT-tests for this?

Reply:
No IT-tests of this part.